api/src/main/java/openconsensus/trace/EndSpanOptions.java

```diff
@@ -25,23 +25,23 @@
  * A class that enables overriding the default values used when ending a {@link Span}. Allows
  * overriding the {@link Status status}.
  *
- * @since 0.1.0
+ * @since 0.1
  */
 @Immutable
 @AutoValue
 public abstract class EndSpanOptions {
   /**
    * The default {@code EndSpanOptions}.
    *
-   * @since 0.1.0
+   * @since 0.1
    */
   public static final EndSpanOptions DEFAULT = builder().build();
 
   /**
    * Returns a new {@link Builder} with default options.
    *
    * @return a new {@code Builder} with default options.
-   * @since 0.1.0
```

Comment:
hm, formatter did it for me. @bogdandrutu do you know why it might happen?

Reply:
I don't think it matters :)